### ISSUE TITLE
add ability to provide a group during a test

### ISSUE THREAD

**Issue by catdad** [catdad/grandma#86]
This can be a method available on `this`, similar to timers, which allows adding any number of groups. This is additive and can be called multiple times. Tentative API:

``` javascript
function test(done) {
  this.group('kiwi');

  this.group(['guava', 'mango']);

  this.group('pineapple', 'orange');

  this.group(['peach', 'pear'], 'blueberry', 'raspberry');
}
```

This will result in all groups being applied, as such:

``` javascript
['kiwi', 'guava', 'mango', 'pineapple', 'orange', 'peach', 'pear', 'blueberry', 'raspberry']
```

**Comment by catdad** [comment]
I might actually want to change this to `category`. Not quite sure yet.

**Comment by catdad** [comment]
Published in 0.6.0.
